examples/aws/api_gateway_with_domain_mtls/aws_gateway_with_domain_mtls.tf

```
@@ -0,0 +1,29 @@
+# --- RESOURCES / MODULES ---
+
+module "lambda_api_gateway_with_domain_mtls" {
+  # source = "github.com/THEY-Consulting/they-terraform//aws/lambda/gateway"
+  source = "../../../aws/lambda/gateway"
+
+  name = "they-test-api-gateway-with-domain_mtls"
+  endpoints = [
+    {
+      path          = "simple"
+      method        = "GET"
+      function_arn  = module.lambda_without_build.arn
+      function_name = module.lambda_without_build.function_name
+    }
+  ]
+
+  domain = {
+    s3_truststore_uri = "s3://they-test-api-gateway-with-domain-assets/certificates/truststore.pem"
+    zone_name          = "they-code.de."
+    domain             = "they-test-gateway-with-mtls.they-code.de"
```

I think this should also be adapted similar as inside with_domain module

```suggestion
    domain             = "${formatdate("YYYY-MM-YY-hh-mm-ss", timestamp())}-they-test-lambda.they-code.de"
```

---

I had the same thought when I added it to dateway_with_domain and then wasn't sure if we needed it. But given that you thought of adding it here, tells me that we do!

---

perhaps we can also just name the domain `${formatdate("YYYY-MM-YY-hh-mm-ss", timestamp())}-they-test.they-code.de"` instead?